src/radical/pilot/raptor/worker_default.py

```diff
@@ -47,12 +47,13 @@ def __init__(self, raptor_id : str):
                                                  cb=self._request_cb)
 
         # the master should have stored our own task description in the registry
-        self._reg.dump('raptor_worker')
-        self._descr = self._reg['raptor.%s.cfg' % self._uid]
+        self._descr = self._reg['raptor.%s.cfg' % self._uid] or {}
```

Comment:
Maybe we should consider it an error if the config is not found?

Reply:
that's related to this https://github.com/radical-cybertools/radical.pilot/pull/3097#issuecomment-1839751426
-> two ways to submit workers, where in one of them worker description is  is not kept in the registry

Reply:
Ack, thanks.